tests/model/test_exif.py

```diff
@@ -27,7 +27,10 @@
      1457, 2084, 1, 1, 1, 'inches', 'RGB', None),
     # tolerate multi-frame TIFF:
     ('gutachten/data/IMG/IMG_1.tif',
-     2088, 2634, 300, 300, 300, 'inches', 'RGB', 'raw')
+     2088, 2634, 300, 300, 300, 'inches', 'RGB', 'raw'),
+    # multi-frame TIFF with metric pixel density (is actually YCBCR not RGB but Pillow thinks otherwise...)
```

Comment:
Actually, Pillow just does not say anything. We could ask ImageMagick about it, though.

(However, `%[EXIF:PhotometricInterpretation]` does not yield anything... Not sure how to get that info.)

So IMO this test should state `YCbCr` and fail until we fix `OcrdExif.run_identify` to include the results from the CLI, or we find a way to make Pillow give us the info.

BTW, using Pillow's [TiffTags](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#tiff) we can actually get the right value:
```python
# val = img.tag_v2.named()['PhotometricInterpretation']
val = img.tag_v2.get(262)
interpretation = dict((v, k) for k, v in TiffTags.TAGS_V2.get(262).enum.items())
val = interpretation[val]
```